sources/Directory.Packages.props

```diff
@@ -1,136 +1,137 @@
-  </ItemGroup>
-  <!-- Windows/WPF dependencies -->
-  <ItemGroup>
-    <PackageVersion Include="AvalonEdit" Version="6.3.0.90" />
-    <PackageVersion Include="Dirkster.AvalonDock" Version="4.72.1" />
-    <PackageVersion Include="Microsoft.Xaml.Behaviors.Wpf" Version="1.1.77" />
-    <PackageVersion Include="Microsoft.Win32.Registry" Version="5.0.0" />
-    <PackageVersion Include="Microsoft-WindowsAPICodePack-Shell" Version="1.1.5" />
-    <PackageVersion Include="Stride.CrashReport" Version="1.0.2" />
-    <PackageVersion Include="Stride.GraphX.WPF.Controls" Version="2.4.0" />
-    <PackageVersion Include="Stride.Metrics" Version="1.0.3" />
-    <PackageVersion Include="RoslynPad.Editor.Windows" Version="1.2.0" />
-    <PackageVersion Include="RoslynPad.Roslyn" Version="3.6.0" />
-    <PackageVersion Include="RoslynPad.Roslyn.Windows" Version="3.6.0" />
-  </ItemGroup>
-  <!-- Analyzers -->
-  <ItemGroup>
-    <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.7.0" PrivateAssets="all" />
-    <PackageVersion Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="all" />
-  </ItemGroup>
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <!-- Runtime dependencies -->
+  <ItemGroup>
+    <PackageVersion Include="FFmpeg.AutoGen" Version="3.4.0.2" />
+    <PackageVersion Include="K4os.Compression.LZ4.Legacy" Version="1.3.6" />
+    <PackageVersion Include="Microsoft.Management.Infrastructure" Version="3.0.0-preview.4" />
+    <PackageVersion Include="Microsoft.NETCore.Platforms" Version="7.0.4" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
+    <PackageVersion Include="ServiceWire" Version="5.5.4" />
+    <PackageVersion Include="SharpDX" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.D3DCompiler" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.Direct2D1" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.Direct3D11" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.Direct3D12" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.DirectInput" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.MediaFoundation" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.RawInput" Version="4.2.0" />
+    <PackageVersion Include="SharpDX.XInput" Version="4.2.0" />
+    <PackageVersion Include="Silk.NET.OpenGL" Version="2.20.0" />
+    <PackageVersion Include="Silk.NET.OpenGLES" Version="2.20.0" />
+    <PackageVersion Include="Silk.NET.OpenGLES.Extensions.EXT" Version="2.20.0" />
+    <PackageVersion Include="Silk.NET.OpenXR" Version="2.20.0" />
+    <PackageVersion Include="Silk.NET.Sdl" Version="2.20.0" />
+    <PackageVersion Include="Silk.NET.Windowing.Sdl" Version="2.19.0" />
+    <PackageVersion Include="sixlabors.imagesharp" Version="3.1.2" />
```

Comment:
The package name might be case-sensitive on Unix platform.

Reply:
In addition it seems the file was saved using a different encoding causing the whole content to show as a diff.